content/projects/kotlin/project-5/data-binding-viewmodel-livedata/_index.md

```diff
@@ -0,0 +1,379 @@
+---
+db_id: 
+available_flavours:
+- kotlin
+content_type: project
+prerequisites:
+  hard:
+  - projects/kotlin/project-5/live-data-observers
+  soft: []
+ready: true
+submission_type: repo
+title: Data binding with ViewModel and LiveData.
+---
+
+### App overview
+
+In the this project, you develop the GuessTheWord app, beginning with starter code. GuessTheWord is a two-player charades-style game, where the players collaborate to achieve the highest score possible.
+
+The first player looks at the words in the app and acts each one out in turn, making sure not to show the word to the second player. The second player tries to guess the word.
+
+To play the game, the first player opens the app on the device and sees a word, for example "guitar," as shown in the screenshot below.
+
+
+### 2. App overview
+
+In the this project, you develop the GuessTheWord app, beginning with starter code. GuessTheWord is a two-player charades-style game, where the players collaborate to achieve the highest score possible.
```

Comment:
This text is repeated inside this file. Rather just introduct the project in one place in one file. The recruits can see the prerequisites whenever they need to. We dont need to repeat ourselves at all.